lib/gds_api/publishing_api_v2.rb

```diff
@@ -315,6 +315,28 @@ def get_content_items(params)
     get_json("#{endpoint}/v2/content#{query}")
   end
 
+  # Returns an Enumerator of content items for the provided
+  # query string parameters.
+  #
+  # @param params [Hash]
+  #
+  # @return [Enumerator] an enumerator of content items
+  #
+  # @see https://github.com/alphagov/publishing-api/blob/master/doc/api.md#get-v2content
+  def get_paged_content_items(params, per_page: 50)
```

Comment:
Also, given that per_page is a parameter that `get_content_items` accepts, it might be more consistent to handle it through the params hash, rather than as a separate argument?